pom.xml

```diff
@@ -498,11 +498,291 @@
       <!-- netty BOM -->
       <dependency>
         <groupId>io.netty</groupId>
-        <artifactId>netty-bom</artifactId>
-        <version>${netty.version}</version>
-        <type>pom</type>
-        <scope>import</scope>
+        <artifactId>netty-buffer</artifactId>
```

Comment:
Any specific need for this pom.xml change?

Reply:
Ah, was a mistake! Removed.